system/security/securesocket/securesocket.cpp

```diff
@@ -774,27 +774,20 @@ void CSecureSocket::readTimeout(void* buf, size32_t min_size, size32_t max_size,
         }
 
         rc = SSL_read(m_ssl, (char*)buf + size_read, max_size - size_read);
```

Comment:
Maybe call ERR_clear_error() before SSL_read() ?

Reply:
Yes ERR_clear_error() should be called before and after. I was thinking not to change the logic for now but do it in a separate issue (a few other places could be added), but I can add it here now.